pegasus/helpers/section_api_helpers.rb

```diff
@@ -113,9 +113,9 @@ def self.completed_levels(user_id)
       and("best_result >= #{ActivityConstants::MINIMUM_PASS_RESULT}")
   end
 
-  VALID_GENDERS = %w(m f)
+  @@valid_genders = %w(m f)
```

Comment:
:facepalm: no, I didn't.  I suppose I was skeptical of the way this file was loaded, but the tests run just fine if I remove the include.